### DISCUSSION TITLE
My Kurzweil K2000 doesn't boot with BlueSCSI connected

### DISCUSSION THREAD

**Discussion by romeczek73** [BlueSCSI/BlueSCSI-v2#177] [Help] [answered]
My Kurzweil K2000R does not boot with BlueSCSI v2 connected. Tested with termination on and off. Please help...

Platform: BlueSCSI
FW Version: 2024.05.21-rel May 21 2024 21:21:44
I2C Supported
Flash chip size: 2048 kB
 
=== SD Card Info ===
SD card detected, FAT32 volume size: 3772 MB
SD Name: SD04G, MID: 0x02, OID: 0x54 0x4D
[150ms]  
[150ms] === Global Config ===
[151ms] Reading configuration from bluescsi.ini
[151ms] Active configuration:
[151ms] -- Debug is enabled
[151ms] DBG -- Parity is enabled
[151ms] DBG -- ReinsertCDOnInquiry is disabled
[192ms]  
[192ms] === Finding images in / ===
[194ms] == Opening /HD10_512  1000 MB.hda for ID: 1 LUN: 0
[409ms] DBG ---- Read prefetch enabled: 8192 bytes
[409ms] ---- Image ready
[409ms] 
[409ms] === ROM Drive ===
[409ms] Platform supports ROM drive up to 1692 kB
[409ms] ---- ROM drive image not detected
[409ms]  
[409ms] === Configured SCSI Devices ===
[409ms] * ID: 1, BlockSize: 512, Type: Fixed, Quirks: None, Size: 1024064kB
[511ms]  
[511ms] Initialization complete!
[705ms] INFO: Pico Voltage: 3.217V.
[707ms] DBG -- BUS_BUSY

**Comment by romeczek73** [comment]
I debugged .ini by myself but still K2000 won't boot

Platform: BlueSCSI
FW Version: 2024.05.21-rel May 21 2024 21:21:44
I2C Supported
Flash chip size: 2048 kB
 
=== SD Card Info ===
SD card detected, FAT32 volume size: 15185 MB
SD Name: SL16G, MID: 0x03, OID: 0x53 0x44
 
=== Global Config ===
Reading configuration from bluescsi.ini
Active configuration:
 
=== Finding images in / ===
== Opening /HD10_512  1000 MB.hda for ID: 1 LUN: 0
---- Image ready

=== ROM Drive ===
Platform supports ROM drive up to 1692 kB
---- ROM drive image not detected
 
=== Configured SCSI Devices ===
* ID: 1, BlockSize: 512, Type: Fixed, Quirks: None, Size: 1024064kB
 
Initialization complete!

**Comment by romeczek73** [comment]
When I'm scanning SCSI ports using K2000's diagnostics it hangs. Seems like I need to try some older BlueSCSIv2 firmware

**Reply by erichelgeson** [reply]
I'm not sure what an older firmware would do but is easy to try. We've had reports of the K2000 working out of the box. 

I'd suggest you go through the entire troubleshooting guide here https://bluescsi.com/docs/Troubleshooting#what-to-try-first to rule out any simple issues.

**Comment by romeczek73** [comment]
Seems like it's something with k2000 because I connected a scsi cdrom and the symptoms are the same

**Comment by romeczek73** [comment, marked as answer]
I found solution on the Internet. SCSI on K2000 is reversed....
I had to take a box cutter and remove the nose from the 50 pin plug and flip it. 
Now everything's working fine :)

**Reply by erichelgeson** [reply]
Can you link to where you found that?

**Comment by romeczek73** [comment]
https://gearspace.com/board/electronic-music-instruments-and-electronic-music-production/1383000-zulu-scsi-scsi2sd-discussion-8.html#post16627924





W dniu 2024-08-25 17:42, Eric Helgeson napisał(a):
